app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryResultsRow.tsx

```diff
@@ -23,13 +23,20 @@ export const ListingDetailsLotteryResultsRow = ({ bucket }: LotteryResultsRowPro
           {bucket.preferenceName === "generalLottery"
             ? t("lottery.generalPool")
             : defaultIfNotTranslated(
-                `listings.lotteryPreference.${bucket.preferenceName}.title`,
-                bucket.preferenceName
+                `listings.lotteryPreference.${bucket.preferenceName
+                  .replace("Tier 1 ", "")
+                  .replace("Tier 2 ", "")}.title`,
+                bucket.preferenceName.replace("Tier 1 ", "").replace("Tier 2 ", "")
```

Comment:
issue: dynamic translation keys need comments per our [guidelines](https://miro.com/app/board/uXjVMzMekug=/?moveToWidget=3458764569463367056&cot=14).

Not written down anywhere but: we have previously discussed how dynamic keys can make debugging etc. harder, and try to avoid it when possible.

Reply:
Ah, thanks I'll add a comment

Reply:
> maybe would be better to have an educator specific ListingDetailsLotteryResultsRow

That's my personal preference (minimizing inline logic), but I'll leave that decision up to you.